### ISSUE TITLE
Image upload add images_file_types param

### ISSUE THREAD

**Issue by bluefatter** [tinymce/tinymce#6607]
Please add images_file_types to init params
https://www.tiny.cloud/docs/configure/file-image-upload/#images_file_types

Current version: 
tinymce 5.7.0
react-tinymce 3.10.2

**Comment by bluefatter** [comment]
I checked again, Editor prop BaseEditorSettings has the type any for additional props. 

But the setting `images_file_types` seems not work in Image upload dialog when I click upload button choose local image.

My code:
![image](https://user-images.githubusercontent.com/20634176/111096979-9dcf1d00-857b-11eb-9d7e-d731df8c1cb9.png)

For test, Base on my code I still can select jpg image in file select dialog

**Comment by JamesToohey** [comment]
Hi @bluefatter,

I will open a PR to fix the missing type for `images_file_types`, thank you for reporting this.

Regarding your other issue, the `images_file_types` setting does not modify the [`accept`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#attr-accept) attribute of the file uploader, which is always set to `'image/*'`. It is only used to filter the files after they have been selected.

This decision was made when we added this setting to ensure we were only ever dealing with images. Allowing custom file extensions here will require more work, so I have marked this as a feature request.
